nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -118,6 +118,8 @@ class ConformerEncoder(NeuralModule, StreamingEncoder, Exportable, AccessMixin):
             Defaults to None.
         conv_dual_mode (bool): specifies if convolution should be dual mode when dual_offline mode is being used. When enables, the left half of the convolution kernel would get masked in streaming cases.
             Defaults to False
+        use_bias (bool): Use bias in all Linear and Conv1d layers from each ConformerLayer to improve activation flow and stabilize training of huge models.
```

Comment:
the comment suggests that we should use bias for training huge models. can you update? (also just `huge` is not informative.. can you update to `huge (>1B)` or the appropriate size)